plugins/view-resources/src/components/filter/ValueFilter.svelte

```diff
@@ -31,6 +31,10 @@
   filter.modes = [view.filter.FilterValueIn, view.filter.FilterValueNin]
   filter.mode = filter.mode === undefined ? filter.modes[0] : filter.mode
 
+  // TODO: remove "TypeString" from search types after
+  // a separate filter for strings is implemented
+  $: isSearchable = [core.class.TypeNumber, core.class.TypeString].includes(filter.key.attribute.type._class)
```

Comment:
We need Enums's here for long categories, but we will loose it. Should we better control it with some mixin?